Sources/ArcGISToolkit/Components/FeatureFormView/FormViewModel.swift

```diff
@@ -90,11 +87,11 @@ import SwiftUI
     /// Performs an evaluation of all form expressions.
     func evaluateExpressions() {
         evaluateTask?.cancel()
-        isEvaluating = true
         evaluateTask = Task {
-            let evaluationErrors = try? await featureForm.evaluateExpressions()
-            expressionEvaluationErrors = evaluationErrors ?? []
-            isEvaluating = false
+            if let evaluationErrors = try? await featureForm.evaluateExpressions(),
+               !evaluationErrors.isEmpty {
+                expressionEvaluationErrors = evaluationErrors
+            }
```

Comment:
What about the case where there previously WERE evaluation errors (`expressionEvaluationErrors` is not empty), but the next call to `evaluateExpressions()` returns an empty array. In that case, the non-empty list of `expressionEvaluationErrors` would not be cleared (because we're only setting it if `evaluationErrors` is not empty).

You would want to update `expressionEvaluationErrors` only if it was not equal to the new list of `evaluationErrors`. I don't know if arrays or `Errors` are `Equatable`...

Reply:
> In that case, the non-empty list of expressionEvaluationErrors would not be cleared.

That's a good point.

> I don't know if arrays or Errors are Equatable...

Arrays should be if their content is but in this case neither `FormExpressionEvaluationError` nor its underlying `Error` is.

Reply:
I see `expressionEvaluationErrors` is not currently being read either. We could consider removing its `@Published` attribute or removing the property all together. Off the top of my head I'm not thinking of where we'd need to use it.

Reply:
Kotlin isn't currently the errors and they don't seem to have a use case presently defined so I'll remove the property for now.

Ref Apollo 271, 280